js/bitstamp.js

```diff
@@ -77,9 +77,53 @@ module.exports = class bitstamp extends Exchange {
                 },
             },
             'fees': {
-                'trading': {
-                    'maker': 0.0025,
-                    'taker': 0.0025,
+                'trading': { // for verified account. Anonymous 0.3 on taker
+                    'tierBased': true, 
+                    'percentage': true,
+                    'taker': [
```

Comment:
can't change these values like that without breaking the other code...

Reply:
ideas to have that working?

Reply:
revert them back to floats and store the fee tiers in the `exchange.fees.tiers` ?